plugins/meta/vrf/vrf.go

```diff
@@ -130,6 +138,14 @@ CONTINUE:
 		}
 	}
 
+	// Apply all saved routes for the interface that was moved to the VRF
+	for _, route := range routes {
+		// modify original table to vrf one
+		// equivalent of 'ip route add <address> table <int>'
+		route.Table = int(vrf.Table)
+		netlink.RouteAdd(&route)
```

Comment:
unhandled error

Reply:
Yes, I noticed it before, but it is not handled by purpose - the error can be ignored, because if adding a route returns error - it can mean that is already there.

Reply:
in that case, to be consistent with the logic of ipv6 addresses which is above this block, only add the routes that do not exist

Reply:
Then check if the error is because the route is already there.  You can't assume that the error will always be due to that. A user might complain that the routes are not being added and we won't have logs to troubleshoot.

Reply:
Ok, I have added a check, if route exists, and if not, return error during adding a route.